### PR TITLE
Update view-active-alarms.md

### PR DESCRIPTION
**Pull request by samrafalowski**

### CODE REVIEW

**Comment by samrafalowski** [comment]
Updating to reflect "alerts" terminology rather than "alarms"

**Comment by ilyam8** [comment]
@samrafalowski 

If this change is a part of an Epic please add a link to this epic, so we see the context of _Updating to use "alerts" term_.

(And please sign the CLA.)

File: docs/monitor/view-active-alarms.md [hunk 1]
```diff
@@ -1,75 +1,75 @@
 <!--
-title: "View active health alarms"
-description: "View active alarms and their rich data to discover and resolve anomalies and performance issues across your infrastructure."
-custom_edit_url: https://github.com/netdata/netdata/edit/master/docs/monitor/view-active-alarms.md
+title: "View active health alerts"
+description: "View active alerts and their rich data to discover and resolve anomalies and performance issues across your infrastructure."
+custom_edit_url: https://github.com/netdata/netdata/edit/master/docs/monitor/view-active-alerts.md
```
Comment (manos-saratsis) [inline docs/monitor/view-active-alarms.md:4]: @samrafalowski the URL is returning me a 404 so I did not approve for now

Comment (samrafalowski) [reply docs/monitor/view-active-alarms.md:4]: Fixed this

**Review by manos-saratsis — APPROVED** [review]

**Comment by ilyam8** [comment]
Closing since this is not finished (there are much more MD files that use the "alarm" word) and is not going to be finished. @manos-saratsis if this is still on our to-do list somebody can take over this PR (or start a new one from scratch).

**Pull request closed without merging**